proxies/events-graphql-proxy/.env.example

```diff
@@ -0,0 +1,10 @@
+NODE_ENV=development
+GRAPHQL_PROXY_DEBUG=debug
+GRAPHQL_PROXY_PORT=4100
+GRAPHQL_PROXY_CMS_API_BASE_URL=https://cms.test.kuva.hel.ninja/api/
```

Comment:
The CMS connection, types and resolvers are now removed, so there should be no need for this. Same thing in `docker-compose.proxy.yml` -file